### PR TITLE
Resolve Issue #10

### LINKED ISSUE #10: Update stylings to Products page
As products and Wishlist page share the same products card, there are some changes that were implemented on the wishlist page that can also be done on Products page.

1. Change height values
2. Add auto margins
3. Apply grid styling properties
4. Apply media queries to dynamically size cards for different screen sizes

### PR DESCRIPTION
**Pull request by Hamdrive**
Referring to issue #10 , changes have been successfully made, and code reviewed and meets expectations.
Submitted for review.

### CODE REVIEW

File: css/products.css [hunk 1]
```diff
@@ -96,4 +117,32 @@
   .my-3 {
     margin: 0;
   }
+
+  .grid-cols-auto {
+    grid-template-columns: 0;
+  }
+
+  .grid-col-1 {
```
Comment (shivampandey0) [inline css/products.css:125]: shouldn't `.grid-col-1` be `.grid-cols-1` to match the consistency of other classes `.grid-cols-auto` and `.grid-cols-2`

**Review by shivampandey0 — COMMENTED** [review]
Looks good.
added one suggestion.

File: css/products.css (hunk 1 above)
Comment (Hamdrive) [reply css/products.css:125]: `grid-col-1` is for creating single column grid, which is singular hence why it is so.

Comment (shivampandey0) [reply css/products.css:125]: okay

**Review by abhij1607 — APPROVED** [review]
Great work, really liked the way of naming classes

**Comment by Hamdrive** [comment]
Issue has been reviewed, this fixes #10 and can be closed now. Merging to dev

**Pull request merged**